Gems/ROS2/Code/Source/RobotImporter/URDF/ArticulationsMaker.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ArticulationsMaker.h"
+#include <AzCore/Component/EntityId.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+#include <RobotImporter/Utils/TypeConversions.h>
+#include <Source/EditorArticulationLinkComponent.h>
+
+namespace ROS2
+{
+    // Here is the recommended, minimal number of iterations for position and velocity solver.
+    // It is needed since currently o3de default values are optimized for the gaming experience, not a simulation.
+    constexpr AZ::u8 MinimalNumPosSolvArticulations = 40;
```

Comment:
it seems these settings are valid whether we are dealing with articulation link or the "traditional" joints. What's more, they might be valid even when adding components by hand when using Robotics Template. 

Perhaps a good first step is to move them out to higher-level header. These could also make sense as registry keys, with different values set for different templates.

Also, considering the name, is "Default" more fitting, instead of "Minimal"?